src/global.css

```diff
@@ -1,17 +1,30 @@
-@import url('https://fonts.googleapis.com/css2?family=Quicksand&display=swap');
-
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@600&family=Roboto:wght@500;900&display=swap");
 *,
 *::after,
 *::before {
   box-sizing: border-box;
 }
+:root {
+  --ff-primary: "Roboto", sans-serif;
+  --ff-secondary: "Montserrat", sans serif;
+  --primary-color: #fe4775;
+  --secondary-color: #161060;
+  --text-primary: #393939;
+  --text-secondary: #111;
+  --content-passive: #d4d4d4;
+  --fw-primary: 400;
+  --fw-primary-light: 300;
+  --fw-primary-regular: 400;
```

Comment:
I think this will fall back to another `font-weight` since these are not imported

Reply:
Thank you! Fixed Font-weight!